### DISCUSSION TITLE
AttributeError: module 'numpy' has no attribute 'int' in Step 6

### DISCUSSION THREAD

**Discussion by AllendePrieto** [alexisInSAR/EZ-InSAR#44] [Help]
Hi Alex,
I have not been able to advance.
I have repeated the whole process and noticed that in step 6 (overlap_resample), I get the error: AttributeError: module 'numpy' has no attribute 'int'.
This is the cause that you cannot generate the interferograms in step 7:
FileNotFoundError: [Errno 2] No such file or directory: '/home/callende/PROCESADO_B/coreg_secondarys/20220605/overlap/IW1_top.xml'.
I can send you results of each of the previous steps if you need it.

![imagen](https://user-images.githubusercontent.com/131383377/236442774-cb82cb17-11c2-42a9-91c3-49a9c09a7939.png)

Thanks for your help!

**Comment by Rick0219** [comment]
I think I know how to fix the problem. You can try  downgraded numpy version.

**Reply by alexisInSAR** [reply]
Thank you.

Let me know if it works.

**Reply by AllendePrieto** [reply]
Ok! Thanks!

**Reply by callendeprieto** [reply]
This appears to correspond to this issue
https://stackoverflow.com/questions/74946845/attributeerror-module-numpy-has-no-attribute-int

and the solution is as simple as replacing numpy.int() by the python intrinsic int().

**Reply by alexisInSAR** [reply]
Thank you. 

Can you discuss this issue with the ISCE2 developers?

**Reply by AllendePrieto** [reply]
Fixed!!

The best solution is to install a downgraded numpy version like Rick wrote.

conda install -c conda-forge numpy=1.21.5

Thanks!
